src/main/kotlin/ar/edu/unahur/obj2/vendedores/centrosDeDistribucion.kt

```diff
@@ -0,0 +1,32 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDeDistribucion(val ciudadActual: Ciudad ) {
+    val vendedoresConLosQueTrabaja = mutableListOf<Vendedor>()
+
+    fun agregarVendedor(vendedor: Vendedor) {
+        if (! this.estaElVendedor(vendedor)) {
+            vendedoresConLosQueTrabaja.add(vendedor)
+        }
+        else {
+            throw Exception("El vendedor ya trabaja con el centro de distribucion")
+        }
```

Comment:
Está bien, aunque queda un poco más prolijo poner el error en la rama verdadera del `if` y obviar el `else`:
```suggestion
        if (this.estaElVendedor(vendedor)) {
            throw Exception("El vendedor ya trabaja con el centro de distribucion")
        }
        vendedoresConLosQueTrabaja.add(vendedor)
```